tango_ros_common/tango_ros_common/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+
+    package="eu.intermodalics.tangoroscommon">
```

Comment:
For consistency reasons shouldn't we call the package "eu.intermodalics.tango_ros_common" ?